client/src/page/MenuPage.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styled from "@emotion/styled";
+import MenuImageSrc from "./../assets/asteronautRakete.png";
+import MenuBackgroundImageSrc from "./../assets/planet.png";
+import { Link } from "react-router-dom";
+
+const MenuWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 100vw;
```

Comment:
I would put this in global styles and give a 100% here

Reply:
Have to look if this get's in the way with my game-page. but I will definitely try. Thanks for the suggestion 😄